package br.org.edu.ifrn.LojaCarro.repository;

import br.org.edu.ifrn.LojaCarro.model.Usuario;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.Optional;
@Repository
public  interface  UsuarioRepository
            extends JpaRepository<Usuario, Long> {

        Optional<Usuario>
        findByUsername(String username);
}
